src/pybind/mgr/dashboard/frontend/src/app/ceph/rgw/rgw-overview-dashboard/rgw-overview-dashboard.component.html

```diff
@@ -112,11 +112,13 @@ <h1>{{ averageObjectSize | dimlessBinary}}</h1>
     <cd-card cardTitle="Multisite Sync Status"
              i18n-title>
       <ng-template #notConfigured>
-        <cd-alert-panel type="info"
-                        i18n>
-          Multisite needs to be configured in order to see the multisite sync status.
-          Please consult the <cd-doc section="multisite"></cd-doc> on how to configure and enable the multisite functionality.
-        </cd-alert-panel>
+        <span class="pe-5 ps-5"> 
+          <cd-alert-panel type="info"
+                          i18n>
```

Comment:
alert-panel has a spacing class
```suggestion
          <cd-alert-panel type="info"  
                          spacingClass='ps-5 pe-5'
                          i18n>
```

Reply:
Didnt work in this case